index.html

```diff
@@ -0,0 +1,169 @@
+        @National Centre Of Confrence, Home Center Oregon, USA.
+      </p>
+    </section>
+    <section class="main-section" id="program">
+      <h3 class="main-title">Main Program</h3>
+      <hr />
+      <ul>
+        <li class="cards">
+          <img src="images/lecturelogo.png" alt="" class="card-pic" />
+          <h4>Online Lectures</h4>
+          <p class="card-about">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+            libero cumque voluptatibus repellat
+          </p>
+        </li>
+        <li class="cards">
+          <img src="images/exhibitionlogo.png" alt="" class="card-pic" />
+          <h4>Exhibitions Worldwide</h4>
+          <p class="card-about">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+            libero cumque voluptatibus repellat
+          </p>
+        </li>
+        <li class="cards">
+          <img src="images/forumlogo.png" alt="" class="card-pic" />
+          <h4>Global Forum</h4>
+          <p class="card-about">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+            libero cumque voluptatibus repellat
+          </p>
+        </li>
+        <li class="cards">
+          <img src="images/workshopicon.png" alt="" class="card-pic" />
+          <h4>Workshop</h4>
+          <p class="card-about">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+            libero cumque voluptatibus repellat
+          </p>
+        </li>
+        <li class="cards">
+          <img src="images/networkingicon.png" alt="" class="card-pic" />
+          <h4>Network</h4>
+          <p class="card-about">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+            libero cumque voluptatibus repellat
+          </p>
+        </li>
+      </ul>
+      <a class="join-title noone"> Join Global Seminar Confrence,2015 </a>
+      <p class="projects none1"><a>COMPLETED PROJECTS</a></p>
+    </section>
+
+    <!-- inject the speakers dynamically with javascript -->
+    <section class="testimonials">
+      <h3 class="main-title">Hear It From Our Speakers</h3>
+      <hr class="speaker-hr" />
+      <div class="grid"></div>
+    </section>
+
+    <section class="sponsors" id="sponsor">
+      <h3>Trusted By</h3>
+      <hr class="about-hr" />
+      <div class="sponsors-logo">
+        <img src="images/mozillaicon.svg" alt="mozilla" />
+        <img src="images/googlelogo.svg" alt="google" />
+        <img src="images/navericon.svg" alt="naver" />
+        <img src="images/daumkakaologo.svg" alt="daumkakao" />
+        <img src="images/airbnblogo.svg" alt="airbnb" />
+      </div>
+    </section>
+    <footer id="join">
+      <img src="images/creative.svg" alt="" />
+      <div class="footright">
+        <p>2019 Town_Square,Washington,USA</p>
+        <!-- <br /> -->
+        <p>All Rights reserved</p>
+      </div>
+    </footer>
```

Comment:
- [ OPTIONAL ] I think the footer section is not in the home page for mobile version it is just in the about page so kindly remove it  from the mobile version it is just displayed in the desktop version so keep it there and make it display:none in the mobile version ✅💪🏻💪🏻

- Your home page
![image](https://user-images.githubusercontent.com/58553711/181521953-b55b2437-fc77-4f40-91c1-b4f916cff72b.png)